modules/tinymce/CHANGELOG.md

```diff
@@ -15,12 +15,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - New `highlight_on_focus` option which enables highlighting the content area on focus. #TINY-9277
 - New `fontsizeinput` toolbar item which allows the user to set the size via input and also increase and decrease it with `+` and `-` buttons. #TINY-9429
 - Added `skipFocus` option to the `ToggleToolbarDrawer` command to preserve focus. #TINY-9337
-- Added `common/space/delBetweenExclamationMarks` to remove spaces between exclamation marks. #TINY-9398
```

Comment:
Where is this one duplicated?

Reply:
> Where is this one duplicated?

As per the *Description of Changes* above:

* Removed entry duplicated from [entry in the Advanced Typography plugin changelog](https://github.com/tinymce/typography/blob/d0a234bbdcea03c15c9307504d1b6dfdbefbcd4a/CHANGELOG.md?plain=1#L11).